inference-engine/ie_bridges/python/tests/test_IECore.py

```diff
@@ -253,3 +256,37 @@ def test_net_from_buffer_valid():
     o_net2 = ref_net.outputs
     assert ii_net.keys() == ii_net2.keys()
     assert o_net.keys() == o_net2.keys()
+
+
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE","CPU") != "GPU", reason=f"Device dependent test")
+def test_load_network_release_gil(device):
+    running = True
+    message_queue = Queue()
+    def detect_long_gil_holds():
+        sleep_time = 0.01
+        latency_alert_threshold = 0.1
+        # Send a message to indicate the thread is running and ready to detect GIL locks
+        message_queue.put("ready to detect")
+        while running:
+            start_sleep = time()
+            sleep(sleep_time)
+            elapsed = time() - start_sleep
+            if elapsed > latency_alert_threshold:
+                # Send a message to the testing thread that a long GIL lock occurred
+                message_queue.put(latency_alert_threshold)
+    ie = IECore()
+    net = ie.read_network(model=test_net_xml, weights=test_net_bin)
+    # Wait for the GIL lock detector to be up and running
+    gil_hold_detection_thread = Thread(daemon=True, target=detect_long_gil_holds)
+    gil_hold_detection_thread.start()
+    # Wait to make sure the thread is started and checking for GIL holds
+    sleep(0.1)
```

Comment:
FYI I think this sleep is no longer necessary, because the message_queue is used to synchronize :grin:

Reply:
> FYI I think this sleep is no longer necessary, because the message_queue is used to synchronize 😁

Without sleep, this test passes without this fix, so i think sleep is necessary